src/pathwayvis/components/legend/legend.component.scss

```diff
@@ -0,0 +1,43 @@
+pv-legend{
+	.legend{
+		margin-left: 2em;
+		margin-bottom: 3em;
+		position: absolute;
+		z-index: 10;
+		background: white;
+		bottom: 0;
+	}
+
+	.grad{
+		background: linear-gradient(-90deg, #A841D0, #868BB2, #6DBFB0, #54B151);
```

Comment:
would make sense to define the colors in one place, where the Escher Builder is, so there is no duplication, but for getting it out faster it'll do
there is a legend in Escher which you probably came across already which could be reused somehow, there are also additional features on it like median and other values https://github.com/zakandrewking/escher/blob/master/js/src/Scale.js